Examples/Tests/electrostatic_sphere_eb/PICMI_inputs_3d.py

```diff
@@ -63,7 +63,7 @@
 embedded_boundary = picmi.EmbeddedBoundary(
     implicit_function="-(x**2+y**2+z**2-radius**2)",
     potential=V_embedded_boundary,
-    radius = 0.3
+    radius = 0.1
```

Comment:
I changed the radius to 0.1 because otherwise the EB was too close to the boundaries of the simulation box, and the results were therefore significantly different from the theoretical expected results.